### PR TITLE
feat: Cairo 0.9.0 support

### PR DESCRIPTION
**Pull request by ca11ab1e**
### What I did

fixes: #17

Tests won't pass on the CI because we first need to merge #16. It is a circular requirements.
I would say to merge that one first, then #16 (if all is good on your side).

### Checklist

- [x] Passes all linting checks (pre-commit and CI jobs)
- [ ] New test cases have been added and are passing
- [ ] Documentation has been updated
- [x] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

File: ape_cairo/compiler.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from ethpm_types import ContractType, PackageManifest
 from pkg_resources import get_distribution  # type: ignore
 from starknet_py.compile.compiler import StarknetCompilationSource, starknet_compile  # type: ignore
-from starkware.starknet.services.api.contract_definition import ContractDefinition  # type: ignore
+from starkware.starknet.services.api.contract_class import ContractClass  # type: ignore
```
Comment (unparalleled-js) [inline ape_cairo/compiler.py:10]: We are going to have to bump `cairo-lang` minimum version requirement to `>=0.9` so that this import succeeds

Comment (unparalleled-js) [reply ape_cairo/compiler.py:10]: Add these changes to `setup.py`:

```
        "cairo-lang>=0.9,<0.10",
        "starknet.py>=0.3.0a0,<0.4.0",
```

NOTE: make sure to first have `ape-starknet` uninstalled. We have upgrade in this order, `ape-cairo` first.

Comment (unparalleled-js) [reply ape_cairo/compiler.py:10]: The tests will pass after this.

Then, we will release `ape-cairo`.
Then we can move on to `ape-starknet`.

Comment (ca11ab1e) [reply ape_cairo/compiler.py:10]: Done :heavy_check_mark:

**Pull request merged**